manual/source/examples/epics/index.rst

```diff
@@ -199,6 +199,25 @@ were arrays rather than single values.  That was due to an error in the original
 This has been fixed in the ``attributes.xml`` file presented here.
 
 
+.. _EPICS_Area_Detector.examples.validate-xml:
+
+Validate the XML files
+**********************
+
+To validate an XML file, you'll need the associated ``.xsd`` (XML Schema) file,
+then use this command::
+
+   xmllint --noout --schema path/to/local/schema/file.xsd  file_to_validate.xml
+
+These are the XML Schema files to use:
```

Comment:
We should also mention `nxdl.xsd`

Reply:
Would you agree that the approach described in this [SO post](https://stackoverflow.com/a/48980995) is a good fit?

In this context, https://www.w3.org/2012/04/XMLSchema.xsd applies to both, with the same command instruction.

Reply:
Validating `nxdlTypes.xsd` this way shows 33 "Schemas validity error" reports. 

> Character content other than whitespace is not allowed because the content type is 'element-only'.

Fails for each unit example element, like this one: https://github.com/nexusformat/definitions/blob/c1225135b59f4b75002551ca5311e2ea3c4b5a6c/nxdlTypes.xsd#L93

This issue should add validation instructions.  Any change in `nxdlTypes.xsd` should come in a separate issue.  It will involve changes to the Python software that creates documentation from this file.

When these .xsd files were written years ago, I created and revised them with a licensed XML-aware editor.  Now, that license has lapsed and I have edited them in VSCode.  These examples of units have missed this type of validation.

Reply:
Also adding instruction for any NXDL file (uses `nxdl.xsd`), for completeness.

Reply:
> Would you agree that the approach described in this [SO post](https://stackoverflow.com/a/48980995) is a good fit?
> 
> In this context, https://www.w3.org/2012/04/XMLSchema.xsd applies to both, with the same command instruction.

Yes, we should include this in the CI too.